charts/pulsar/values.yaml

```diff
@@ -1029,6 +1029,7 @@ broker:
     annotations: {}
   offload:
     enabled: false
+    # Mark this flag as deprecated and remove it in the future, it is recommended to use the enabled flag to decide whether to enabled or disabled offload
     use: true
```

Comment:
if we've removed all presence of this value from Chart, I don't think we need to keep it anymore